src/components/titlePane/createTitlePane.ts

```diff
@@ -0,0 +1,88 @@
+import { VNodeProperties } from '@dojo/interfaces/vdom';
+import createWidgetBase from '@dojo/widget-core/createWidgetBase';
+import { v } from '@dojo/widget-core/d';
+import { DNode, Widget, WidgetFactory, WidgetProperties } from '@dojo/widget-core/interfaces';
+import themeable, { ThemeableMixin } from '@dojo/widget-core/mixins/themeable';
+
+import * as css from './styles/titlePane.css';
+import * as animations from '../../styles/animations.css';
+
+export interface TitlePaneProperties extends WidgetProperties {
+	collapsed?: boolean;
+	collapsible?: boolean;
+	enterAnimation?: string;
+	exitAnimation?: string;
+	title: string;
```

Comment:
Fair. In that case, there's no need to default it when you destructure in `getChildrenNodes`.